_15thnight/core.py

```diff
@@ -95,10 +96,26 @@ def respond_to_alert(provider, needs_provided, alert):
     return response
 
 
-def send_out_resolution(need):
-    """
-    Sends out a resolution for a need to providers.
-    """
+def send_alert_resolution_notice(need):
+    # Check if alert is closed and trigger an alert closed notification
+    if not need.alert.is_open():
+        alert = need.alert
+        body = "Alert sent at {0} for a {1}yo " % (alert.created_at, alert.age)
+        body += "{0} " % alert.gender if alert.gender.lower() in [
```

Comment:
Close, can be done simpler with:
`gender = '' if alert.gender == 'unspecified' else alert.gender`